src/components/CustomCarousel/CustomCarousel.jsx

```diff
@@ -0,0 +1,74 @@
+import Carousel from 'react-multi-carousel';
+import styles from "./CustomCarousel";
```

Comment:
this style import doesn't look similar to all other style imports

Reply:
I'll fix it

Reply:
done